html-css/portfolio.css

```diff
@@ -0,0 +1,136 @@
+
+body {
+    font-family: Arial, sans-serif;
+    color: #333;
+    background-color: #fafafa;
+    line-height: 1.6;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+nav ul {
```

Comment:
wiser to have hover effects on your navigation bar so make the necessary adjustment.